### ISSUE TITLE
please integrate some binaural beat sounds

### ISSUE THREAD

**Issue by firefoxlover** [CliffracerMerchant/SoundAura#28]
Here are some links where you can get CC and royalty free sounds:

https://orangefreesounds.com/binaural-beats/
https://archive.org/details/binaural-beats_201904
https://musicalhypnosis.com/royalty-free-binaural-beats/

Awesome app! This is so useful, but needs some basic samples without background music for mixing.

**Comment by CliffracerMerchant** [comment]
I originally decided to leave out pre-included sounds to cut down on download size. I'm not opposed to including a few if they are small enough and aren't too obvious when they loop, but it might have to wait for the time being unless someone else wants to do it. I have quite a few other open issues at the moment, and I'm not going to have nearly as much time to work on this app in the near future.

I'll leave the issue open for now in case I ever get around to it, or in case someone else wants to handle it.

**Comment by sanctimoniousape** [comment]
I like your mindset to keep the app small, but it does leave the user a bit too much on their own for those not knowing where to go for sound sources (yeah, I know Google exists, but good app design makes the barriers to entry as low as possible, IMHO).  As a simple solution, may I suggest you include an intro dialog saying something like "Don't know where to start? Check out these sites for some sound options." Include the above links, along with any others you'd like.  That at least gives the user a good starting point.  The lack of one is why I came here, as I was hoping to find some recommended sources - thanks to @firefoxlover for fulfilling that need.

**Comment by CliffracerMerchant** [comment]
I think I'm still opposed to actually including the sounds in the apk, but you're right that there should be a pop up dialog to point users in the right direction. I'll add one that links to the [FreeSound.org ambient sounds section](https://freesound.org/browse/tags/ambience/), and a pointer to the [Seal app](https://f-droid.org/en/packages/com.junkfood.seal/) (which can be used to download audio from websites like YouTube.

**Comment by firefoxlover** [comment]
Good idea! Never heard of Seal, there are no many downloaders. Also dont forget [Spotiflyer](https://f-droid.org/en/packages/com.shabinder.spotiflyer/).

I would divide it into categories:

Ambient sounds
- [Freesounds.org](https://freesound.org/browse/tags/ambience/)

Binaural Beats
- [Orangefreesound](https://orangefreesounds.com/binaural-beats/) (CC)
- [Gehirnkicker](https://www.gehirnkicker.de/binaural-beats-kostenlos/#) (free)
- [Pixabay](https://pixabay.com/music/search/binaural%20beats/) ([Own License](https://pixabay.com/service/license-summary/), lots of Javascript)


music
- souces

**Comment by PawelGu** [comment]
Yeah I think there are enough sound sources. Personally I like much more to use YouTube as a around source - I prefer non looping sounds so I combine 2 or 3 hours recordings of a beach and tropical forest sound...
What would be nice though having an option for automatically changing volume or pausing of tracks while playing (changing either just randomly or within certain amounts of time). This would lead to more variations.

**Comment by firefoxlover** [comment]
Youtubes Audio quality is pretty bad if you ask certain people. Also they dont offer surround sound, not always stereo etc.

**Comment by heartlog** [comment]
You know what's better than annoying pop ups? A nice markdown file that tells you everything you need to know about the sound sources. Just create a file called "soundsources.md" and write down the websites and notes about them. It's like a cheat sheet for your ears. And if you want to be extra fancy, you can also use the "repo/wiki/" section to store your information. Trust me, the simpler you make it, the easier it will be to understand it. And the more people will love you for it.

**Comment by Jonas-fr** [comment]
I'd rather have a minimalist app (lightweight) so adding sounds by myself isn't an issue for me and it's even a plus because I don't want to bloat my device with unwanted files/sounds. If anyone wants some a pack of nicely loopable sounds here are the ones I use : https://archive.org/details/soundaura_sounds

There's just a small gap when they loop which is a bug, see this issue : https://github.com/CliffracerMerchant/SoundAura/issues/64
There's also an issue with really short ones which get annoying as they loop really fast. I used them in a past with an app which allowed randomized play of them (like the airplane ping, which would play once in a given time period, which was configurable). I opened an issue for that : https://github.com/CliffracerMerchant/SoundAura/issues/65

**Comment by Robio** [comment]
I'm with the developer here. I'd much rather have a tiny app and fetch my own sound files.
